scripts/py.test.sh

```diff
@@ -1,18 +1,12 @@
 
```

Comment:
@nelson-liu do we even use this for anything anymore?  I don't think we do, especially now that just running `py.test` from the shell does everything you need.